### ISSUE TITLE
TripsLayer data doesn't' fill up from async iterable

### ISSUE THREAD

**Issue by zsolt-truemotion** [uber/deck.gl#3976]
I have an async iterable function that yields batches from a local array `mockData` which contains a list with the well formatted trips objects:

```javascript
const mockData = [
{
sensors: [[-95.36685, 29.75864], [-95.36683, 29.7592], [-95.36702, 29.75932], ...],
ts: [51992.5, 51993, 51998.1, ...]
},
{
sensors: [[-95.41291, 29.73832], [-95.41291, 29.73832], [-95.41291, 29.73832], ...],
ts: [51078.2, 51082.1, 51091.6, 51100.4,...]
},
...
];
```

The async iterable function looks like this:
```javascript
async* getData() {
    for (let i = 0; i < mockData.length; i++) {
      await this._delay(100);
      yield [mockData[i]];
    }
 }
```

And the TripsLayer: 
```javascript
 new TripsLayer({
        id: 'trips',
        data: getData(),
        getPath: d => d.sensors,
        getTimestamps: d => d.ts,
        currentTime: 52000,
})
```

I'm not getting any error's although the trips layer doesn't show up.
If I change the `data: getData()` to `data: mockData` it works fine.
Other layer's I tries (scatterPlot, lineLayer) worked also fine with the provided async iterable.

Thanks,
Zsolt

**Comment by Pessimistress** [comment]
What is the version you are using? We patched a bug in loading data from async iterables in 7.3.5.

**Comment by zsolt-truemotion** [comment]
Hey, thanks for your reply. I'm using version 7.3.6

**Comment by Pessimistress** [comment]
A bug is patched in 7.3.8. https://codepen.io/Pessimistress/pen/vYEKGQM?editors=0010
